### PR TITLE
Initialize Custom MUI Theme

### PR DESCRIPTION
**Pull request by rcamach7**
I still need to fill in with custom color fields and other changes we want to make. We can keep the PR open until we figure out theme colors and customizations

### CODE REVIEW

File: client/src/theme/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    secondary: {
+      main: "#C4055E",
+      dark: "#890341",
+      light: "#cf377e",
+    },
+    /** TODO: Select custom colors or remove
+     * You would be able to use it like this: `<Button color="custom">`
+     */
+    custom: {
```
[the remark this answers is not in the document]
Comment (lugenx) [reply client/src/theme/index.js:19]: @rcamach I think this is the only thing left before merge for this PR.

[the remark this answers is not in the document]
Comment (lugenx) [reply client/src/theme/index.js:19]: @rcamach7 I think this is the only thing left before merge for this PR.

**Comment by rcamach7** [comment]
Ok let me know if any other changes are needed

**Review by lugenx — APPROVED** [review]
great work, thank you!

**Pull request merged**